### PR TITLE
Update CHANGELOG-0.1.4.md

### PR DESCRIPTION
**Pull request by juchi**

### CODE REVIEW

File: app/Resources/doc/changeLog/CHANGELOG-0.1.4.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ Url to see changes :
  - [Theme bundle](https://github.com/open-orchestra/open-orchestra-theme-bundle/compare/v0.1.3...v0.1.4)
 
 ## Possible BC breaker
-
+- The ContactController in the DisplayBundle has been removed. Therefore there are no more controller in
```
Comment (nicolasThal) [inline app/Resources/doc/changeLog/CHANGELOG-0.1.4.md]: I would rather put this line in the configuration change section

Comment (juchi) [inline app/Resources/doc/changeLog/CHANGELOG-0.1.4.md]: Well, i tought so, but :
* it's clearly a BC breaker
* it's not a configuration key that changed, it just may impact your configuration

Comment (nicolasThal) [inline app/Resources/doc/changeLog/CHANGELOG-0.1.4.md]: All right, so you could split in two informations

**Comment by juchi** [comment]
ping @nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**